python/cudf/cudf/_libxx/aggregation.pyx

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import pandas as pd
+import numba
+import numpy as np
+from libcpp.string cimport string
+from cudf.utils import cudautils
+
+from cudf._libxx.lib cimport *
+
+from cudf._libxx.column import np_to_cudf_types
+from cudf._libxx.aggregation cimport *
+
+# need to update as and when we add new aggregations with additional options
+cdef unique_ptr[aggregation] get_aggregation(op, kwargs):
+
+    cdef type_id tid
+    cdef data_type out_dtype
+    cdef string cpp_str
+    cdef unique_ptr[aggregation] agg
+
+    if op is "sum":
```

Comment:
We should use `==` when checking for string equality as opposed to `is`.

`is` checks if the LHS and RHS are the same object, which for strings is often not the case:

```python
>>> a = "sum"
>>> b = "sum"
>>> a is b
True
>>> a == b
True
>>> a = "sum_foo"
>>> a[:3] is b
False
>>> a[:3] == b
True
```


```